interfaces/builtin/docker_support.go

```diff
@@ -175,8 +175,22 @@ ptrace (read, trace) peer=cri-containerd.apparmor.d,
 # needed by runc for mitigation of CVE-2019-5736
 # For details see https://bugs.launchpad.net/apparmor/+bug/1820344
 / ix,
+/bin/busybox ixr,
```

Comment:
is this also for mitigation of CVE-2019-5736 ?

if not, maybe add a space between the above `/ ix` and this to delineate the different accesses?

Reply:
Based on the denial (`[runc:INIT]`) it looked to be in the same vein as pause, so I put it there. I can move pause and busybox a line below, sure.

Reply:
Done.